plugins/lookup/passwordstore.py

```diff
@@ -386,11 +394,11 @@ def update_password(self):
         # generate new password, insert old lines from current result and return new password
         newpass = self.get_newpass()
         datetime = time.strftime("%d/%m/%Y %H:%M:%S")
-        msg = newpass + '\n'
-        if self.passoutput[1:]:
-            msg += '\n'.join(self.passoutput[1:]) + '\n'
-        if self.paramvals['backup']:
-            msg += "lookup_pass: old password was {0} (Updated on {1})\n".format(self.password, datetime)
+        msg = newpass
+        if self.paramvals['preserve'] and self.passoutput[1:]:
+            msg += '\n' + '\n'.join(self.passoutput[1:])
```

Comment:
Before, if `self.passoutput[1:]`, a newline was added here at the end, even if `self.paramvals['backup'] == False`. This no longer happens.

Reply:
You're right, but I don't know if that's the right behavior. In my case, I store multiple line passwords in a file, from which I would then have to remove this character (newline).

This is the solution that comes to mind:
```python
if self.paramvals['preserve']:
    msg += '\n' 
    if self.passoutput[1:]:
        msg += '\n'.join(self.passoutput[1:])
```

What do you think?

Reply:
How about add it if `self.paramvals['preserve'] or self.paramvals['timestamp']`? That would also allow to keep the `msg` modifications in the two `if`s unchanged.

Reply:
It sounds good. So this is the final proposal...?
```python
if self.paramvals['preserve'] or self.paramvals['timestamp']:
    msg += '\n' 
    if self.passoutput[1:]:
        msg += '\n'.join(self.passoutput[1:])
```

Reply:
Shouldn't it be
```py
if self.paramvals['preserve'] or self.paramvals['timestamp']:
    msg += '\n' 
    if self.paramvals['preserve'] and self.passoutput[1:]:
        msg += '\n'.join(self.passoutput[1:])
```
?

Reply:
I agree.